aip/general/0122.md

```diff
@@ -294,6 +293,8 @@ equivalent to the corresponding message's name in snake case.
 - Fields representing another resource **should** provide the
   `google.api.resource_reference` annotation with the resource type being
   referenced.
+- If using the resource name is not possible and using the ID component alone is strictly necessary, the field **should** use an `_id` suffix (e.g.
```

Comment:
Formatting: Need to wrap this line at 80 chars

Reply:
done!